src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -2203,6 +2335,41 @@ type Plan {
     debugOutput: debugOutput!
 }
 
+type PlanConnection {
+    searchDateTime: PlanDateTime
+
+    locations: PlanPlaces
+
+    routingErrors: [RoutingError!]!
+
+    debugInformation: PlanDebugInformation!
+
+    edges: [PlanEdge]
+
+    pageInfo: PageInfo!
+}
+
+type PlanDateTime {
+    date: DateTime
+    timeZone: TimeZone
+}
+
+type PlanDebugInformation {
+    totalTime: Int
+    pathCalculationTime: Int
```

Comment:
I think we can debate if a Duration is a good idea here. 200ms will be represented as `PT0.2S`. On balance, I think I like it but I can see that not everyone is of the same opinion.

Reply:
Yes I'm not sure about that. Also in more general I wonder if parsing these durations is as easy for clients than just using Int.

Reply:
We can discuss this in dev meeting